### ISSUE TITLE
[Carousel A11y] Unable to navigate to elements inside carousel using keyboard

### ISSUE THREAD

**Issue by JeanRoca** [microsoft/AdaptiveCards#7661]
https://microsoft.visualstudio.com/OS/_workitems/edit/39442334
